app/controllers/application_controller.rb

```diff
@@ -1,5 +1,17 @@
 class ApplicationController < ActionController::Base
-  def self.current_user
-    User.first
+  # def self.current_user
+  #   User.first
+  # end
```

Comment:
- [ ] Kindly remove these commented codes to achieve a clean codebase.

Reply:
This issue is well resolved! ✔️